lib/semantic-toast-container.jsx

```diff
@@ -62,8 +63,15 @@ class SemanticToastContainer extends Component {
     };
 
     updateToasts = () => {
+        // Add the new toast data to state.
         this.setState({
             toasts: store.data
+        }, ()=> {
```

Comment:
The usage of the `setState` callback is not considered a good practice. If your are not debugging something, then perhaps hooking into `componentDidUpdate()` is the proper way to do it.

Reply:
Good idea, moved the logic to [2fd32cc](https://github.com/academia-de-codigo/react-semantic-toasts/pull/63/commits/2fd32ccbc343590bc471515381f61c07172e456f)